Cabal-hooks/Cabal-hooks.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version: 2.2
 name:          Cabal-hooks
-version:       3.14
+version:       3.14.2

```

Comment:
I was under the impression that this was deliberately being left without a minor version.

Reply:
```suggestion
version:       3.14

```